cmd/flags.go

```diff
@@ -255,6 +255,11 @@ var (
 		Usage: "Specifies the timeout value for API requests in seconds",
 		Value: 120,
 	}
+	// JwtOutputFileFlag specifies the JWT file path that gets generated into when invoked by generate-jwt-secret.
+	JwtOutputFileFlag = &cli.StringFlag{
+		Name:  "output-file",
```

Comment:
`output-file` feels a little vague as a cli flag name

Reply:
because it's used in this CLI command, that has no other flags, having a clear name for the output file is preferred. Many other CLIs use this pattern, or even just have `-o` as a flag, which is more intuitive. We can also add shorthand -o